src/website/app/demos/Card/index.js

```diff
@@ -28,35 +28,56 @@ const cardBlockDoc = require('!!react-docgen-loader!../../../../Card/CardBlock')
 const cardImageDoc = require('!!react-docgen-loader!../../../../Card/CardImage');
 const cardTitleDoc = require('!!react-docgen-loader!../../../../Card/CardTitle');
 
+import bestPractices from './bestPractices';
+
 export default [
   {
+    bestPractices: bestPractices.card,
     componentTheme: cardComponentTheme,
     doc: cardDoc,
     examples: cardExamples,
     slug: 'card',
-    title: 'Card'
+    title: 'Card',
+    whenHowToUse: `Cards are used when you need to group different kinds of elements with actions attached.
+Cards are used to contain content of variable length that might not fit neatly into a table row or grid cell.
+They're often used to show a collection of related but different types of information.
+
+Don't put too much information in a single card.
+Cards represent a gateway to more detailed information in another app view.`
   },
   {
+    bestPractices: bestPractices.cardBlock,
     componentTheme: cardBlockComponentTheme,
     doc: cardBlockDoc,
     examples: cardBlockExamples,
     slug: 'card-block',
     subcomponent: true,
-    title: 'CardBlock'
+    title: 'CardBlock',
+    whenHowToUse: `CardBlock is used to help lay out content that's not a [title](./card-title) or an [image](./card-image) in the body of the [Card](./card).
+
+Try not to put inline links in your content. Create purposeful calls to action with [Buttons](./button) at the bottom of the Card.`
   },
   {
+    bestPractices: bestPractices.cardImage,
     doc: cardImageDoc,
     examples: cardImageExamples,
     slug: 'card-image',
     subcomponent: true,
-    title: 'CardImage'
+    title: 'CardImage',
+    whenHowToUse: `CardImage is used when you want to reinforce the intent of the Card.
+Images probably shouldn't be used alone in a Card, but should be paired with a call to action and/or a CardTitle.
+
+If you are putting text over top of the CardImage, use a solid color or an image with sufficient contrast to the text.`
   },
   {
+    bestPractices: bestPractices.cardTitle,
     componentTheme: cardTitleComponentTheme,
     doc: cardTitleDoc,
     examples: cardTitleExamples,
     slug: 'card-title',
     subcomponent: true,
-    title: 'CardTitle'
+    title: 'CardTitle',
+    whenHowToUse: `Use a [CardTitle](./card-title) when you need a consistently styled headings for your [Card](./card).
```

Comment:
CardTitle shouldn't be a link here; they're already on that page if they're reading this content.

Reply:
ah yes, good catch.